requirements.txt

```diff
@@ -0,0 +1,5 @@
+cookiecutter>=1.4.0
+
+# Testing
+pytest==3.0.5
+pytest-cookies==0.2.0
```

Comment:
Do you think we should have separate files for different requirements?

requirements_dev.txt
requirements_run.txt
??